bbs/admin/unix/install.sh

```diff
@@ -326,14 +331,22 @@ then
   fi
 else
   say "User ${WWIV_USER} not found; creating it."
-  useradd -g ${WWIV_GROUP} -c "WWIV BBS Service Account" -d ${WWIV_DIR} \
-	  -s /sbin/nologin -m ${WWIV_USER} >> ${LOGFILE} 2>&1
+  if [[ $OSTYPE =~ "SunOS" ]]
+  then
```

Comment:
what about setting the logon shell based on OSTYPE, then we don't have to duplicate the whole useradd line